nuget.config

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
 	<packageSources>
-		<clear/>
```

Comment:
Do you get prompted for creds every time you open  a project? I suspect this clear was part of the problem. It's unclear though. I'd love to track this down.